src/ResourceManager/ResourceManager/Commands.ResourceManager/Cmdlets/Implementation/GetAzureResourceCmdlet.cs

```diff
@@ -139,6 +139,7 @@ public sealed class GetAzureResourceCmdlet : ResourceManagerCmdletBase
         [Parameter(ParameterSetName = GetAzureResourceCmdlet.ListResourcesParameterSet, Mandatory = false, HelpMessage = "The name of the tag to query by.")]
         [Parameter(ParameterSetName = GetAzureResourceCmdlet.MultiSubscriptionListResourcesParameterSet, Mandatory = false, HelpMessage = "The name of the tag to query by.")]
         [ValidateNotNullOrEmpty]
+        [Obsolete("This parameter is obsolete and will be removed in future releases.")]
```

Comment:
Do not add this - it will actually make the build fail, because we *have* top turn on treat warnings as errors.  Instead, print a messag to the verbose stream when the command is executed if a user uses this parameter